src/image-copy/azext_imagecopy/__init__.py

```diff
@@ -43,6 +43,8 @@ def load_arguments(self, _):
                        help='Name of the final image that will be created')
             c.argument('target_subscription', options_list=['--target-subscription'],
                        help='Name or ID of the subscription where the final image should be created.')
+            c.argument('time_out', options_list=['--timeout'], type=int, default=3600,
```

Comment:
I think the config parameter and the internal variable name could be the same, can't it?

Reply:
@afeng what about this one?

Reply:
@afengli what about this one?